tests/ui/await_holding_invalid.stderr

```diff
@@ -32,32 +32,125 @@ LL | | }
    | |_^
 
 error: this MutexGuard is held across an 'await' point. Consider using an async-aware Mutex type or ensuring the MutexGuard is dropped before calling await.
-  --> $DIR/await_holding_invalid.rs:41:13
+  --> $DIR/await_holding_invalid.rs:42:13
    |
 LL |         let guard = x.lock().unwrap();
    |             ^^^^^
    |
 note: these are all the await points this lock is held through
-  --> $DIR/await_holding_invalid.rs:41:9
+  --> $DIR/await_holding_invalid.rs:42:9
    |
 LL | /         let guard = x.lock().unwrap();
 LL | |         baz().await
 LL | |     };
    | |_____^
 
 error: this MutexGuard is held across an 'await' point. Consider using an async-aware Mutex type or ensuring the MutexGuard is dropped before calling await.
-  --> $DIR/await_holding_invalid.rs:53:13
+  --> $DIR/await_holding_invalid.rs:54:13
    |
 LL |         let guard = x.lock().unwrap();
    |             ^^^^^
    |
 note: these are all the await points this lock is held through
-  --> $DIR/await_holding_invalid.rs:53:9
+  --> $DIR/await_holding_invalid.rs:54:9
    |
 LL | /         let guard = x.lock().unwrap();
 LL | |         baz().await
 LL | |     }
    | |_____^
 
-error: aborting due to 4 previous errors
+error: this RefCell Ref is held across an 'await' point. Consider ensuring the Ref is dropped before calling await.
```

Comment:
It would be nice to keep the tests separated.

Reply:
Done. Unfortunately, I don't think I was able to preserve the git history when they were split. I could attempt to squash the commit where I restore the split into the commit where I merged the two files.